### DISCUSSION TITLE
How do I add custom Helm charts to an EKS cluster?

### DISCUSSION THREAD

**Discussion by josh-padnick** [gruntwork-io/knowledge-base#576] [How do I] [answered]
Is there a recommended way to add a custom Helm chart to an EKS cluster so that it's managed as infrastructure as code? For example, I'd like add RabbitMQ via Helm. Separately, are there any standard Helm charts you recommend?

---

<ins datetime="2022-10-21T23:36:29Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109416">Tracked in ticket #109416</a></p>
</ins>

**Comment by autero1** [comment, marked as answer]
To deploy arbitrary Helm Charts, you can use the [Helm Chart wrapper module](https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/modules/services/helm-service) in Service Catalog. That module is intended to deploy third-party applications already packaged as Helm Charts. 

[Bitnami](https://bitnami.com/stacks/helm) has a fairly wide variety of applications packaged as Helm Charts.
